src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.TestService/Helpers.cs

```diff
@@ -4,15 +4,14 @@
 
 namespace System.ServiceProcess.Tests
 {
-    [Flags]
     public enum PipeMessageByteCode
     {
-        Start = 0,
-        Continue = 1,
-        Pause = 2,
-        Stop = 4,
-        OnCustomCommand = 8,
-        ExceptionThrown = 16,
-        Connected = 32
+        Start,
+        Continue,
+        Pause,
+        Stop,
+        OnCustomCommand,
+        ExceptionThrown,
+        Connected
```

Comment:
Why doesn't this need to be a [Flags] anymore?

Reply:
When I added this, I thought I would be using the flag attribute.but I end up not using this. so removing it in this PR